ocs_ci/ocs/constants.py

```diff
@@ -921,13 +921,11 @@
 RDR_MODE = "regional-dr"
 MDR_MODE = "metro-dr"
 MDR_DR_POLICY = "odr-policy-mdr"
-MDR_RESTIC_POD_COUNT = 3
-MDR_VELERO_POD_COUNT = 1
+RESTIC_POD_COUNT = 3
+VELERO_POD_COUNT = 1
 MDR_DPA = "dpa-1"
-MDR_MULTICLUSTER_ENGINE = "multiclusterengine"
-MDR_BACKUP_SCHEDULE_YAML = os.path.join(
-    TEMPLATE_MULTICLUSTER_DIR, "backupschedule.yaml"
-)
+MULTICLUSTER_ENGINE = "multiclusterengine"
+BACKUP_SCHEDULE_YAML = os.path.join(TEMPLATE_MULTICLUSTER_DIR, "backupschedule.yaml")
```

Comment:
backupschedule.yaml needs changes, raised- https://github.com/red-hat-storage/ocs-ci/issues/9554

Reply:
This will be taken in different PR